docs/cow-protocol/concepts/order-types/pay-debt-flash-loans.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 6
+---
+
+# Repay debt with collateral using flash loans
```

Comment:
Shouldn't the order type section be titled just 'Flash Loans,' with debt repayment described inside as the common use case? Or do we consider flash loans in CoW to be used only for this specific purpose?